### PR TITLE
Helama bio

### PR DESCRIPTION
**Pull request by helaman-Y-B**
hello i'm helama

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # cse210-tc03
-CSE210 activities
```
Comment (helaman-Y-B) [inline README.md:2]: Put it back

**Comment by helaman-Y-B** [comment]
Take this helama.

**Review by helaman-Y-B — COMMENTED** [review]
added some stuff

**Comment by fireburn553** [comment]
I don't the conflict

**Pull request merged**